app/javascript/components/shared_components/Header.jsx

```diff
@@ -13,6 +13,17 @@ export default function Header() {
   const deleteSession = useDeleteSession();
   const { data: brandingImage } = useSiteSetting('BrandingImage');
 
+  const adminAccess = () => {
+    const { permissions } = currentUser;
+    delete permissions.SharedList;
+    delete permissions.CreateRoom;
+
+    if (Object.values(permissions).includes('true')) {
+      return <NavDropdown.Item as={Link} to="/adminpanel">Admin Panel</NavDropdown.Item>;
+    }
+    return false;
```

Comment:
This is trying to be too smart - what happens if we add another permission unrelated to admin? 

When it comes to security, always manually check for trues, rather than remove the unneeded permissions and then checking blindly for true